staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2668,7 +2674,10 @@ const (
 // The requirements of them are ANDed.
 // It provides a subset of functionality as NodeSelectorTerm.
 // This is an alpha feature and may change in the future.
+// +structType=atomic
 type TopologySelectorTerm struct {
+	// Usage: Fields of type []TopologySelectorTerm must be listType=atomic.
```

Comment:
There are `AllowedTopologies []v1.TopologySelectorTerm` uses that are not marked as listType=atomic... where will that be done?

Reply:
Commented on Joe's PR earlier, listType is atomic by default so omitting the annotation doesn't change any behavior. Would you prefer it to be explicitly marked?

Reply:
I'm personally happy with explicitly marking it as atomic